// 処理系のテスト４６
// 異常ケース３５: 同じ名前の手続きを宣言

void still(int x);
void move(int x);
void still(int x);	// still() は既に宣言済みなのでエラー

void still(int x)
{
  int log = x - 100;
}

void move(int x)
{
  int log = x - 100;
}

void main()
{
  move(10);
}

